### DISCUSSION TITLE
Flameshot alternative for Linux

### DISCUSSION THREAD

**Discussion by OnkelTem** [flameshot-org/flameshot#3480] [Q&A] [answered]
Hi folks.

Could you please share some good alternatives to Flameshot?

I really loved Flameshot and it was one of my favorite programs ever created. Unfortunately I cannot get it working anymore. I tried deb, snap and AppImage - w/o any success. So I'm looking for a program with similar functionality. Specifically I seek for a tool where I can:

1) capture screen regions
2) draw arrows, squares, obfuscate text
3) upload to the Internet, save to file and clipboard.

Thanks in advance.

**Comment by mmahmoudian** [comment, marked as answer]
Don't you think it is inappropriate to post such thing in our github repo?

Although this is highly inappropriate, you can try [ksnip](https://github.com/ksnip/ksnip).

**Reply by OnkelTem** [reply]
Thanks for the suggestion, Mehrad, I'll check it out.

--

As for the question, it was unexpected for me, really. Sorry if it offended you in some way, let me share my attitude on this.

Flameshot is GPL, so if one wished to make software even more Free, they would have probably failed.

Free software is not a capitalistic market, where hiding information is a regular way to compete. Often times Github projects list similar software on their **front pages**, — exactly to give people a choice if for some reason they need to check something else out. But there is no such list in this repo, hence - this topic. 

Flameshot is IMO the best software for the task, but since I couldn't get it working for more than a week being on the most popular LTS Linux distro, I now have to search for alternatives.

**Reply by mmahmoudian** [reply]
There are tons of options out there, but none of the do what Flameshot does. They all allow the user to annotate the screenshot after it was taken and in a windowed manner. Therefore, there are no highly similar alternatives.

Regarding of the part that you couldn't get it working, as I responded in the other issue you opened, Flameshot runs flawlessly on X11. 90% of existing runtime issues are Wayland related, which is a headache of its own.


These said, Flameshot and Ksnip have a good history and relationship. Both project have helped one another in the past, and the dev of Ksnip is a very intelligent and reliable person.

**Reply by OnkelTem** [reply]
I've finally got flameshot working! But only the flatpak version. Neither AppImage nor Snap seem to be functional on my Kubuntu 22.04.

AppImage - doesn't work with imgur, host not found + cannot copy images to the clipboard if launched via KDE shortcut.
Deb - outdated and doesn't support uploading
Snap - ...I forgot why it didn't work sorry
